turbopack/crates/turbopack-tests/tests/snapshot/basic-tree-shake/dynamic-import/output/4c35f_tests_snapshot_basic-tree-shake_dynamic-import_input_lib_b2d8c81e.js

```diff
@@ -11,7 +11,7 @@ __turbopack_context__.s({
         (new_dog)=>dog = new_dog
     ])
 });
-let dog = "dog";
```

Comment:
Why aren't there any snapshot failures? The files aren't identical anymore to the ones generated by Turbopack

Reply:
this is the updated snapshot for the diff in `turbopack/crates/turbopack-tests/tests/snapshot/basic-tree-shake/dynamic-import/input/lib.js`, right?

Reply:
all the snapshots were updated in this commit

https://github.com/vercel/next.js/commit/2e0e417e9b90b090be9e394a9e20e5f8b4eb5ced

Reply:
Yes, but I wonder why the snapshot test pass with these changes. Because clearly, the snapshots aren't identical anymore to the ones written by Turbopack.

Reply:
discussed in slak: The surprising thing was the quotation marks since we expected them to be normalized in some way, but it turns out that SWC doesn't do that.